### DISCUSSION TITLE
cant get wildcards to work

### DISCUSSION THREAD

**Discussion by ghost** [adieyal/sd-dynamic-prompts#283] [Q&A] [answered]
what the title says, ive tried lots of things and the only thing that seems to activate wildcards is putting Max generations on 0. any help with getting it to normally work? ill include screenshots

![image](https://user-images.githubusercontent.com/126641485/222149077-e9b2eda1-d1d8-4c49-9865-8d3da1716fc3.png)
![image](https://user-images.githubusercontent.com/126641485/222149166-55915d0c-b014-4fda-b095-241573d92ccf.png)

**Comment by akx** [comment, marked as answer]
Your prompt has `__body__` etc. in it, but the wildcards are in the `Cameron` subdirectory, e.g. `__Cameron/body__`. You'll need to use `__Cameron/body__` and so on.
